### PR TITLE
VDB-1480 add msg sender to pot cage

### PR DESCRIPTION
**Pull request by elizabethengelman**

### CODE REVIEW

**Review by paytonrules — APPROVED** [review]
Does this one need an integration test?

**Comment by elizabethengelman** [comment]
Good question @paytonrules - the pot cage integration test is currently pending, since one of those events hasn't occurred yet. I may add the assertion in there, just so we don't forget if we ever are able to un-pend this test.

**Review by yaoandrew — APPROVED** [review]
Looking good 🚢

File: transformers/test_data/pot_cage.go [hunk 1]
```diff
@@ -38,10 +38,11 @@ var PotCageEventLog = core.EventLog{
 var potCageModel = event.InsertionModel{
 	SchemaName:     constants.MakerSchema,
 	TableName:      constants.PotCageTable,
-	OrderedColumns: []event.ColumnName{event.HeaderFK, event.LogFK},
+	OrderedColumns: []event.ColumnName{event.HeaderFK, event.LogFK, constants.MsgSenderColumn},
 	ColumnValues: event.ColumnValues{
 		event.HeaderFK: PotCageEventLog.HeaderID,
 		event.LogFK:    PotCageEventLog.ID,
+		//constants.MsgSender
```
Comment (yaoandrew) [inline transformers/test_data/pot_cage.go:45]: 🎉

**Pull request merged**